AkitaInteractive/Hachiko-Engine
# Fix/when destroying crystals half of the screen

To Test:

Go to AnimationPlayer scene and you will see two cristals, i know that they have the same color, when you accept the pr I will change the scene prefabs for these new ones as well as the textures. TY
